### PR TITLE
Ignore mDNS candidates for relay policy

### PR DESCRIPTION
**Pull request by youennf**
Fixes https://github.com/rtcweb-wg/mdns-ice-candidates/issues/122

### CODE REVIEW

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 1]
```diff
@@ -269,16 +277,18 @@ is used:
 ".local" or if the value contains more than one ".", then process the candidate
 as defined in {{RFC8445}}.
 
-2. Otherwise, resolve the candidate using mDNS. The ICE agent SHOULD set the
+2. If the ICE agent candidate policy is "relay", as defined in {{JSEP}}, ignore the candidate.
```
Comment (tQsW) [inline draft-ietf-rtcweb-mdns-ice-candidates.md:280]: nit: ICE candidate policy (using the exact term as in JSEP)

**Review by tQsW — APPROVED** [review]

File: draft-ietf-rtcweb-mdns-ice-candidates.md (hunk 1 above)
Comment (youennf) [inline draft-ietf-rtcweb-mdns-ice-candidates.md:280]:
```suggestion
2. If the ICE candidate policy is "relay", as defined in {{JSEP}}, ignore the candidate.
```

Comment (youennf) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: Fixed

**Review by juberti — COMMENTED** [review]
LGTM

**Pull request merged**